stl/inc/stacktrace

```diff
@@ -300,16 +273,51 @@ public:
     }
 
 private:
-    static constexpr size_t _Max_frames = 0xFFFF;
+    // <NOTICE> Currently vector is enhanced as unconditionally noexcept move-assignable and swappable
 
     static constexpr bool _Noex_move = allocator_traits<_Alloc>::propagate_on_container_move_assignment::value
-                                    || allocator_traits<_Alloc>::is_always_equal::value;
+                                    || allocator_traits<_Alloc>::is_always_equal::value
+                                    || is_nothrow_move_assignable_v<_Frames_t>; // strengthened
+
+    // <NOTICE> ?_Try_frames may be subject to further decreasements?
+
+    static constexpr size_type _Try_frames = 128; // Sufficient in most cases.

```

Comment:
I think it's better to make `_Try_frames` a constexpr variable in `_Current` as it is only related to implementation details.

( or in the contending impl in the link, make it just a pure value with a comment for it: )
```cpp
// First try small _Limit, then try _Max_frames
for (size_type _Limit: {128, _Max_frames}) {
    ....
}
```